# Camera intrinsic matrix parameters (OpenCV)
camera:
  #fx: 462.1379699707031
  #fy: 462.1379699707031
  #cx: 320.0
  #cy: 240.0
  fx: 544.771755
  fy: 546.966312
  cx: 322.376103
  cy: 245.357925
  width: 640
  height: 480

semantic_pcl:
  color_image_topic: "/camera/color/image_raw"
  depth_image_topic: "/camera/depth/image_rect_raw"
  #color_image_topic: "camera/color/image_raw"
  #depth_image_topic: "/camera/depth/image_raw"
  point_type: 1
  #frame_id: "/camera_color_optical_frame"
  frame_id: "/camera_color_optical_frame"
